core/services/validators_test.go

```diff
@@ -543,23 +543,27 @@ schemaVersion      = 1
 contractAddress    = "0x613a38AC1659769640aaE063C651F48E0250454C"
 p2pPeerID          = "12D3KooWHfYFQ8hGttAYbMCevQVESEQhzJAqFZokMVtom8bNxwGq"
 p2pBootstrapPeers  = [
-   "/dns4/chain.link/tcp/1234/p2p/16Uiu2HAm58SP7UL8zsnpeuwHfytLocaqgnyaYKP8wu7qRdrixLju",
+"/dns4/chain.link/tcp/1234/p2p/16Uiu2HAm58SP7UL8zsnpeuwHfytLocaqgnyaYKP8wu7qRdrixLju",
 ]
 isBootstrapPeer    = true
 keyBundleID        = "73e8966a78ca09bb912e9565cfb79fbe8a6048fab1f0cf49b18047c3895e0447"
 monitoringEndpoint = "chain.link:4321"
 transmitterAddress = "0xaA07d525B4006a2f927D79CA78a23A8ee680A32A"
 observationTimeout = "10s"
 observationSource = """
-   ds1          [type=bridge name=voter_turnout];
-   ds1_parse    [type=jsonparse path="one,two"];
-   ds1_multiply [type=multiply times=1.23];
-   ds1 -> ds1_parse -> ds1_multiply -> answer1;
-   answer1      [type=median index=0];
+ds1          [type=bridge name=voter_turnout];
+ds1_parse    [type=jsonparse path="one,two"];
+ds1_multiply [type=multiply times=1.23];
+ds1 -> ds1_parse -> ds1_multiply -> answer1;
+answer1      [type=median index=0];
```

Comment:
Did the library change require the whitespace change?

Reply:
nope